deps/wazuh_testing/wazuh_testing/logcollector.py

```diff
@@ -761,3 +761,28 @@ def format_macos_message_pattern(process_name, message, type='log', subsystem=No
             macos_message = f"{process_name}\[\d+\]: Created Activity ID.* Description: {message}"
 
     return macos_message
+
+def compose_macos_log_command(type="", level="", predicate="", is_sierra=False):
```

Comment:
```suggestion
def compose_macos_log_command(type, level, predicate, is_sierra=False):
```

Reply:
These parameters are optional, in case they are not used they must be empty strings